package.json

```diff
@@ -60,10 +60,10 @@
     "@jbx-protocol/juice-v1-token-terminal": "1.0.1",
     "@jbx-protocol/project-handles": "2.0.2",
     "@jbx-protocol/ve-nft": "0.0.6",
-    "@lingui/cli": "^3.13.0",
-    "@lingui/detect-locale": "^3.13.0",
-    "@lingui/macro": "^3.13.0",
-    "@lingui/react": "^3.13.0",
+    "@lingui/cli": "^3.15.0",
```

Comment:
bumped packages because the change log looked like they made a bunch of useful bug fixes.